pkg/sql/drop.go

```diff
@@ -427,6 +432,16 @@ func (p *planner) dropIndexByName(
 	return nil
 }
 
+func (p *planner) removeSchemaChangerForMutation(mutationID sqlbase.MutationID) {
```

Comment:
You need to find whether the column or indexes is being added through one of the outstanding schema changers. This method would then remove that schema changer and return true